notebooks/eval_notebook.ipynb

```diff
@@ -0,0 +1,721 @@
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Generating train split:   0%|          | 0/1087583 [00:00<?, ? examples/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "6abe7fc18d824d7f8a035122340dd401",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Generating validation split:   0%|          | 0/10982 [00:00<?, ? examples/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from delphi.eval import constants\n",
+    "\n",
+    "tokenized_corpus_dataset = cast(Dataset, load_dataset(\n",
+    "    constants.tokenized_corpus_dataset,\n",
+    "    split=split\n",
+    "))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Processing model llama2-100k\n",
+      "Processing model llama2-200k\n",
+      "Processing model llama2-400k\n",
+      "Processing model llama2-800k\n",
+      "Processing model llama2-1.6m\n",
+      "Processing model llama2-3.2m\n",
+      "Processing model llama2-6.4m\n",
+      "Processing model llama2-12.8m\n",
+      "Processing model llama2-25.6m\n"
+     ]
+    }
+   ],
+   "source": [
+    "from delphi.eval.calc_model_group_stats import calc_model_group_stats\n",
+    "\n",
+    "\n",
+    "token_groups = {id:{'selected':id in selected_ids} for id in vocab.values()}\n",
+    "model_group_stats = calc_model_group_stats(\n",
+    "    list(tokenized_corpus_dataset), next_logprobs_plot, token_groups, ['selected']\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b72e05f2a72d481bba57e94fd9e5540d",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "VBox(children=(Dropdown(description='Token Category:', options=('selected',), value='selected'), FigureWidget(…"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from collections import defaultdict\n",
+    "from delphi.eval.vis_per_token_model import visualize_per_token_category\n",
+    "\n",
+    "performance_data = defaultdict(dict)\n",
+    "for model in models:\n",
+    "    stats = model_group_stats[(model, 'selected')]\n",
+    "    performance_data[model]['selected'] = (\n",
+    "        -stats[\"median\"],\n",
+    "        -stats[\"75th\"],\n",
+    "        -stats[\"25th\"],\n",
+    "    )\n",
+    "\n",
+    "# TODO: this is using the older version of the plotting func\n",
+    "visualize_per_token_category(\n",
```

Comment:
Is it possible to "hook" the token selector in the UI for this, and use the [panel integration with ipywidgets](https://panel.holoviz.org/reference/panes/IPyWidget.html) to display? This would make the thing interactive and have the selected tokens separate from the visualization below.

I'm not certain what to do if we want to have a shared token select for all of the visualization though.